src/api_manager/quota_control.cc

```diff
@@ -28,16 +28,15 @@ namespace api_manager {
 
 void QuotaControl(std::shared_ptr<context::RequestContext> context,
                   std::function<void(Status status)> continuation) {
-  std::shared_ptr<cloud_trace::CloudTraceSpan> trace_span(
```

Comment:
Same comment about publicly documented behavior changing

Reply:
I will like to have correct trace spans. I will assume users will not have scripts to parse the spans automatically so such changes will not be a breaking changes.